### ISSUE TITLE
perTouch replaced by perSwipe?

### ISSUE THREAD

**Issue by chris-82** [glidejs/glide#593]
Hi,

I just updated my glidejs to 3.5.2 and found out that the "perTouch" option doesn't work (exist) anymore. I now found the "perSwipe" option, but when showing a 50 slides slider with 10 slides "perView", I can only decide to slide one-by-one or all 10; there is no compromise in between to keep it flexible with the "perTouch: false" option. Or am I missing something?

Best regards
Chris

**Comment by Angel-Vieira** [comment]
Hey, Were you able to resolve this issue? I was making a website using this framework and I encountered the same problem, I can't use perTouch

**Comment by jeffdrumgod** [comment]
Yes, documentation is apparently out of date, but the attribute is actually `perSwipe` which does the same thing as described in the documentation as `perTouch`
